docs/app/getting-data-with-pelican/mcp/page.mdx

```diff
@@ -0,0 +1,333 @@
+# Using Pelican with AI Assistants via MCP
+
+The Pelican Model Context Protocol (MCP) server enables AI assistants like Claude Code, VS Code Copilot, and other MCP-compatible tools to download files, inspect metadata, and list directories from Pelican federations. This allows you to use natural language to interact with Pelican data directly from your AI-powered development environment.
+
+## What is MCP?
+
+The [Model Context Protocol](https://modelcontextprotocol.io/) is an open standard that allows AI assistants to connect to external tools and data sources. By running a Pelican MCP server, your AI assistant can work with **public** namespaces (not support protected namespaces):
```

Comment:
"not support protected namespace" --> "this does not support protected namespaces"